CleverTap/Runtime/Native/UnityNativeWrapper/Models/UnityNativeBaseEventQueue.cs

```diff
@@ -26,13 +26,14 @@ internal abstract class UnityNativeBaseEventQueue
 
         protected UnityNativeCoreState coreState;
         protected UnityNativeNetworkEngine networkEngine;
-
+        protected UnityNativeEventValidator eventValidator;
         private Coroutine timerCoroutine;
 
-        internal UnityNativeBaseEventQueue(UnityNativeCoreState coreState, UnityNativeNetworkEngine networkEngine, int queueLimit = 49, int defaultTimerInterval = 1)
+        internal UnityNativeBaseEventQueue(UnityNativeCoreState coreState, UnityNativeNetworkEngine networkEngine,UnityNativeEventValidator eventValidator, int queueLimit = 49, int defaultTimerInterval = 1)
```

Comment:
eventValidator is not needed since it is unused.

Reply:
Removed